readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -0,0 +1,79 @@
+# Read the Docs configuration file
+
+# The version of the spec to be use
+version: enum('2')
+
+# Formats of the documentation to be built
+# Default: ['htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia']
+formats: any(list(enum('htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia')), enum('all'), required=False)
+
+# The path to the requirements file from the root of the project
+requirements_file: str(required=False)
+
+# Configuration for Conda support
+conda: include('conda', required=False)
+
+# Configuration for the documentation build process
+build: include('build', required=False)
+
+# Configuration of the Python executable to be used
+python: include('python', required=False)
+
+# Configuration for sphinx
+sphinx: include('sphinx', required=False)
+
+# Submodules configuration
+submodules: include('submodules', required=False)
+
+# Redirects for the current version to be built
+redirects: include('redirects', required=False)
+
+---
+
+conda:
+  # The path to the Conda environment file from the root of the project
+  file: str()
+
+build:
+  # The build docker image to be used
+  # Default: '2.0'
+  image: enum('1.0', '2.0', 'latest')
```

Comment:
I think this is `required=False`, isn't it?

Reply:
Yes and not, the `required=False` is already here https://github.com/rtfd/readthedocs.org/pull/4084/files#diff-bd10ec86a98c7d6bdf7ea8455b8fdfd4R17. But in case we want to add more options here it will be needed.

Reply:
Yeah, I saw the line you pointed. I think we should make it explicit here to avoid future issues. Otherwise, when we add a new option here we will need to remember to add the `required=False` in this line also... which we may forget to do.